backend/backend/settings.py

```diff
@@ -128,3 +135,54 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+# CORS (Cross Origin Resource Sharing)
+
+CORS_ALLOWED_ORIGINS = [
```

Comment:
Just make this true for all.  Since we will have a different url for the frontend after deploying. I think the code was CORS_ORIGIN_ALLOW_ALL = TRUE